app/models/rtd_location_creator.rb

```diff
@@ -0,0 +1,16 @@
+class RtdLocationCreator
+
+  def create_record(rtd_locations_data)
+    parsed_data = JSON.parse(rtd_locations_data)
+    parsed_data.map do |data|
+      data[1].map do |location|
+        parsed_data = JSON.parse(location, symbolize_names: true)
+
+        city = data[0]
+        name = parsed_data[:name]
```

Comment:
Kelly mentioned briefly to me about how the city is called differently than the actual location's data. Great work on figuring that out!!